docs/stories/Stack.stories.mdx

```diff
@@ -4,13 +4,13 @@ import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 
 import { Stack, Box } from '@strapi/design-system';
 
+import { DeprecationNotice } from '../components/DeprecationNotice';
+
 <Meta title="Design System/Technical Components/Stack" component={Stack} />
 
 # Stack
 
-This is the doc of the `Stack` component
-
-[View source](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/Stack)
+<DeprecationNotice href="/?path=/docs/design-system-technical-components-flex--base">Flex</DeprecationNotice>
```

Comment:
Should we also mention that it can be replaced by using the `gap` prop?

Reply:
random thought about `gap` I had this morning, if we pass a single value to gap it'll set `row-gap` and `column-gap` with this value - if we pass an array then it's still handled as one single value but adding responsive capabilities
I don't think it can properly replace `spacing` of `Stack` right now as it doesn't allow to customize row and column values
if I'm not making a mistake there, should we consider improving that? (for `Grid` and `Flex` btw)

Reply:
Doesn't gap follow the flex-direction? If set to row it will add a gap between columns and if set to column it will add a gap between the rows?

The fact that it is not capable of handling responsive values wouldn't worry me too much, because basically nothing is responsive ...

Reply:
mmh for flex you're right - for Grid though it will apply both directions
not really worried about responsive on this one, as you said nothing is responsive rn 😬 just that we don't offer granularity on gap (at least for Grid)

Reply:
I agree, but we are not deprecating `Grid`, because it is way more complex than Flex and can handle different cases :)

Reply:
>Should we also mention that it can be replaced by using the gap prop?

Maybe i'm being too literal, but using the `gap` prop does not change it to `flex`? We can add migration guides for each component if you'd like.

Reply:
> Doesn't gap follow the flex-direction? If set to row it will add a gap between columns and if set to column it will add a gap between the rows?

But you can pass two values as the CSS properties so it's something we should think about in the future:

https://codesandbox.io/s/vigilant-sanderson-c4vihv?file=/src/styles.css